src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -162,7 +161,7 @@
   (let [limit-clause     (if limit (format " LIMIT %s" limit) "")
         offset-clause    (if offset (format " OFFSET %s" offset) "")
         order-by-clause  (order-by->sql order_by)]
-    (format "SELECT paged_results.* FROM (%s) paged_results%s%s%s"
+    (format "SELECT paged_results.* FROM (%s %s%s%s) paged_results"
             sql order-by-clause limit-clause offset-clause)))
```

Comment:
No doubt, this pagination stuff is a great place we can consider since it could collapse one entire level of nesting, but yeah agreed this is for later :-). It was on my list when I first added honeysql, but fell out of scope near the end.